File: Iniciante/URI 2930.r
input = file('stdin', 'r')

line = strsplit(readLines(input, n=1), " ")
e = as.integer(line[[1]][1])
d = as.integer(line[[1]][2])

if (e > d) {
    write("Eu odeio a professora!", '')
} else if ((d - e) >= 3) {
    write("Muito bem! Apresenta antes do Natal!", '')
} else if ((e + 2) < 24) {
    write("Parece o trabalho do meu filho!\nTCC Apresentado!", '')
} else {
    write("Parece o trabalho do meu filho!\nFail! Entao eh nataaaaal!", '')
}

File: Iniciante/URI 2930.php
<?php
    $line = explode(" ", trim(fgets(STDIN)));
    $E = floatval($line[0]);
    $D = floatval($line[1]);
    
    if ($E > $D) {
        echo"Eu odeio a professora!\n";
    } else if (($D - $E) >= 3) {
        echo"Muito bem! Apresenta antes do Natal!\n";
    } else if (($E + 2) < 24) {
        echo"Parece o trabalho do meu filho!\nTCC Apresentado!\n";
    } else {
        echo"Parece o trabalho do meu filho!\nFail! Entao eh nataaaaal!\n";
    }
?>